modules/launcher/src/main/java/org/mule/runtime/module/launcher/MuleContainer.java

```diff
@@ -21,7 +22,10 @@
 import static org.mule.runtime.core.internal.logging.LogUtil.log;
 import static org.mule.runtime.module.deployment.internal.MuleDeploymentService.findSchedulerService;
 import static org.mule.runtime.module.deployment.internal.processor.SerializedAstArtifactConfigurationProcessor.serializedAstWithFallbackArtifactConfigurationProcessor;
+import static com.mulesoft.mule.runtime.module.cluster.internal.security.NodeSecurityManager.PROPERTY_SECURITY_MODEL;
+import static com.mulesoft.mule.runtime.module.cluster.internal.security.NodeSecurityManager.FIPS_SECURITY_MODEL;
```

Comment:
We need to decide if FIPS support needs to be added to Mule CE or just Mule EE.

If it is Mule CE, we would have to define this constants somewhere in this project, because we can't create a dependency on Mule EE.

If it is only Mule EE then, check if all of this can be done in the [MuleContainerBootstrap.java](https://github.com/mulesoft/mule-ee/blob/master/modules/boot/src/main/java/org/mule/runtime/module/reboot/MuleContainerBootstrap.java) from `mule-ee`.